src/30_Advanced/SeatMap_Multiple_Selection.html

```diff
@@ -0,0 +1,155 @@
+    <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+    <style amp-custom>
+      :root {
+        --color-primary: #005AF0;
+        --color-secondary: #00DCC0;
+        --space-3: 1.5rem; /* 24px */
+        --space-4: 2rem;   /* 32px */
+      }
+      .seatmap-container {
+        position: relative;
+        height: 50vh;
+        min-width: 100%;
+        box-sizing: border-box;
+      }
+      .seatmap-container [placeholder] svg {
+        height: 100%;
+      }
+      .seatmap-container .disabled {
+        pointer-events: none;
+        cursor: default;
+        opacity: 0.5;
+        filter: alpha(opacity=50);
+      }
+      .seatmap-container .seat {
+        stroke-width: 3;
+        stroke: rgb(0,0,0);
+        fill:white;
+      }
+      .seatmap-container .unavailable {
+        fill: var(--color-secondary);
+        pointer-events: none;
+      }
+      .seatmap {
+        align-items: flex-start;
+      }
+      .seatmap svg {
+        box-sizing: border-box;
+        padding-top: var(--space-4);
+      }
+      .seatmap svg rect[selected].seat,  .seatmap svg rect.selected.seat{
+        fill: var(--color-primary);
+      }
+      .seatmap [option][selected] {
+        outline: none;
+      }
+      .seatmap >  amp-selector {
+          left: 0;
+          right: 0;
+          margin: auto;
+      }
+      .seatmap  amp-selector .svg-container {
+        padding: 0 var(--space-3);
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style>
+    <noscript>
+      <style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>
+    </noscript>
+
+  </head>
+  <body>
+    <!-- ## Seatmap state -->
+    <!-- Here we are using an `amp-state` called `seatMultiSelection` to keep all the useful information for the seat selection.
+      * we load a list of neighbours seats from the server
+      * `selectedSeats` contains a list of selected seat ids
+      * `numberOfSeats` contains the number of seats we want automatically select -->
+    <amp-state id="seatMultiSelection" src="/json/seatmap-neighbours.json">
+      <script type="application/json">
+      {
+      "selectedSeats": [],
+      "numberOfSeats": 3
+      }
+      </script>
+    </amp-state>
+
+    <!-- ## Seatmap with multiple selection implementation -->
```

Comment:
suggestion: remove `implementation` from the title

Reply:
done